laterpay-migrator/asset_sources/scss/objects/_lists.scss

```diff
@@ -0,0 +1,13 @@
+.lp_list,
+%lp_list,
+.lp_list__item {
+    margin: 0;
+    padding: 0;
+}
+
+.lp_list--bulleted {
+    @extend %lp_list;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Name of placeholder `lp_list` should be written in all lowercase letters with hyphens instead of underscores